drbg/vectors/Readme.md

```diff
@@ -0,0 +1,4 @@
+# NIST Test vectors for SP 800 90Ar1 Hash DRBG
+
+The script `get_vectors.sh` is meant to be run from this path and not from
+root. It will fetch the test vectors from NIST.
```

Comment:
We should mention that we already have test vectors available (checked in). But in order to refresh these - in case there is an updated data set, this script can be executed to update the Hash_DRBG.txt in folders "no_reseed", "pr_false", and "pr_true"